### PR TITLE
ENG-1119: allow to supply pure json inputs to session

### PR DESCRIPTION
**Pull request by itayd**

### CODE REVIEW

File: internal/backend/sessionsgrpcsvc/svc.go [hunk 1]
```diff
@@ -39,6 +41,29 @@ func (s *server) Start(ctx context.Context, req *connect.Request[sessionsv1.Star
...
+		if err := d.Decode(&x); err != nil {
+			err = sdkerrors.NewInvalidArgumentError(`json_inputs["%s"]: %w`, k, err)
+			return nil, sdkerrors.AsConnectError(err)
+		}
+
+		if msg.Session.Inputs == nil {
+			msg.Session.Inputs = make(map[string]*sdktypes.ValuePB)
+		}
+
+		v, err := sdktypes.WrapValue(x)
```
Comment (ig-ra) [inline internal/backend/sessionsgrpcsvc/svc.go:58]: do we need to warp it to unwrap later?

File: proto/autokitteh/sessions/v1/svc.proto [hunk 2]
```diff
@@ -13,6 +13,9 @@ message StartRequest {
   };
 
   Session session = 1 [(buf.validate.field).required = true];
+
+  // Helper: if set, overwrites the session's inputs.
```
Comment (ig-ra) [inline proto/autokitteh/sessions/v1/svc.proto]: if set, *merge and possibly override* the session inputs?

File: internal/backend/sessionsgrpcsvc/svc.go (hunk 1 above)
Comment (itayd) [reply internal/backend/sessionsgrpcsvc/svc.go:58]: it's coming in unwrapped. we need to wrap it.

File: proto/autokitteh/sessions/v1/svc.proto (hunk 2 above)
Comment (itayd) [reply proto/autokitteh/sessions/v1/svc.proto]: done

**Review by daabr — APPROVED** [review]

**Pull request merged**